examples/alerting_example/public/alert_types/always_firing.tsx

```diff
@@ -51,6 +51,7 @@ export function getAlertType(): AlertTypeModel {
       }
       return validationResult;
     },
+    isUiEditEnabled: true,
```

Comment:
Could this simply be `isEditable` or is that ambiguous due to other aspects being editable?